improver/threshold.py

```diff
@@ -274,6 +270,24 @@ def _decode_comparison_operator_string(self):
             )
             raise ValueError(msg)
 
+    def _update_metadata(self, cube):
+        """Rename the cube and add attributes to the threshold coordinate
+        after merging
+        """
+        threshold_coord = cube.coord(self.threshold_coord_name)
+        threshold_coord.attributes.update(
+            {"spp__relative_to_threshold": self.comparison_operator["spp_string"]}
+        )
+        if cube.cell_methods:
+            format_cell_methods_for_probability(cube, self.threshold_coord_name)
+
+        cube.rename(
+            "probability_of_{}_{}_threshold".format(
+                self.threshold_coord_name, probability_is_above_or_below(cube)
```

Comment:
Small nitpick, feel free to ignore - could this be a f-string? For consistency (I note f-string is elsewhere in this PR) and because they can be easier to read.

Reply:
I think this will be harder to read in this case because the length of the variables will make the string very long. A good compromise would be to use keywords instead:
```suggestion
            "probability_of_{parameter}_{relative_to}_threshold".format(
                parameter=self.threshold_coord_name,
                relative_to=probability_is_above_or_below(cube)
# OR
            f"probability_of_{self.threshold_coord_name}_{probability_is_above_or_below(cube)}_threshold"
```

Reply:
Fair point, this one is not so readable in f-string, keyword option looks good.

Reply:
OK, will modify that now.

Reply:
@lucyleeow we may need to merge this today as it interacts with some other urgent work.  So hopefully you are OK with the changes, but feel free to comment on the PR afterwards if not, and I can create a follow up.  Thanks for your review.

Reply:
LGTM thanks!